README.md

```diff
@@ -37,15 +39,26 @@
 
 ### API Documentation
 
-- The updated API documentation is created with [Postman](https://www.postman.com/) and can be found [here](https://documenter.getpostman.com/view/24955418/2sA2xmUqFr).
+- The autogenerated OpenAPI/Swagger documentation is found at server `/api` endpoint.
+  - Fully intractable testing interface
```

Comment:
``` Suggestion
  - Fully interactable testing interface
```
Should this say interactable? Intractable seems to mean "hard to control or deal with".